nuget.config

```diff
@@ -1,13 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <disabledPackageSources>
```

Comment:
wasn't this required a little while ago?

Reply:
Yea it was in two different places so I am just removing the extra one.

Reply:
i think clear needs to be on top

---
In reply to: [581489335](https://github.com/microsoft/ProjectReunion/pull/473#discussion_r581489335) [](ancestors = 581489335)

Reply:
https://www.1eswiki.com/wiki/How_to_Securely_Configure_Package_Source_Files

hmm in the example, it's at the bottom

Reply:
never mind. i was talking about clear at line 4.

---
In reply to: [581569746](https://github.com/microsoft/ProjectReunion/pull/473#discussion_r581569746) [](ancestors = 581569746)